package.json

```diff
@@ -7,7 +7,9 @@
     "format": "prettier --write '**/*.{js,vue,css,scss}'",
     "pretest": "DB_CONNECTION=testing php artisan migrate:fresh && DB_CONNECTION=testing php artisan db:seed",
     "posttest": "vendor/bin/phpstan analyse && vendor/bin/psalm",
-    "test": "vendor/bin/phpunit"
+    "test": "vendor/bin/phpunit",
+    "docker:build": "docker build --cache-from monica-next -t monica-next -f scripts/docker/Dockerfile . && docker image prune -f",
```

Comment:
`docker image prune -f` will prune all images, it's not necessary in this case

Reply:
It prunes only <none> images, I found it useful in terms of cleanup.

Reply:
OK, make sense.